portal/src/util/projectname.test.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * @jest-environment jsdom
```

Comment:
Refer to the documentation of Jest and set the environment to jsdom **globally**

Reply:
https://github.com/authgear/authgear-server/pull/2266/commits/5f22fcd77bd10f87e29d02014a9e903268319b16